examples/industry-coefficient.rs

```diff
@@ -42,7 +45,7 @@ fn main() -> db_dump::Result<()> {
         if row.date >= start_date && row.date < max_date {
             let crate_id = versions[&row.version_id];
             let downloads = downloads.entry(crate_id).or_insert_with(Downloads::default);
-            match row.date.weekday() {
+            match row.date.date().weekday() {
```

Comment:
I tried to check the relevant issues in chrono but I didn't find why `Day<Tz>` can't have its own `Datelike` impl like I'd expected in https://github.com/chronotope/chrono/issues/820#issuecomment-1312807339. `weekday()` has an obvious meaning on a `Day<Tz>` as do all the other methods in https://docs.rs/chrono/0.4.23/chrono/trait.Datelike.html.

Reply:
I've now implemented that over at #883 and updated this to utilize the `Datelike` impl.